### PR TITLE
Solve AOP problem

### PR DESCRIPTION
**Pull request by alextimonov**
Make registration an Expression classes in Spring container, advising all public methodes, delete all Operation classes

### CODE REVIEW

File: AppCalc/src/main/java/ua/goit/timonov/enterprise/AppConfig.java [hunk 1]
```diff
@@ -9,52 +9,59 @@
  * Java-based Spring configuration class for AppCalc
  */
 
-@Configuration
-@EnableAspectJAutoProxy
-@Import(LogAspect.class)
+//@Configuration
```
Comment (schaffe) [inline AppCalc/src/main/java/ua/goit/timonov/enterprise/AppConfig.java:12]: why comments?

File: AppCalc/target/appCalc.log [hunk 2]
```diff
@@ -1,11 +1,132 @@
-08-07-2016 16:21:44 INFO  AnnotationConfigApplicationContext:581 - Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@3b764bce: startup date [Fri Jul 08 16:21:44 EEST 2016]; root of context hierarchy
```
Comment (schaffe) [inline AppCalc/target/appCalc.log:1]: don`t commit logs

**Comment by schaffe** [comment]
Remove binary files, logs from this pull request

File: AppCalc/src/main/java/ua/goit/timonov/enterprise/AppConfig.java (hunk 1 above)
Comment (alextimonov) [inline AppCalc/src/main/java/ua/goit/timonov/enterprise/AppConfig.java:12]: OK, I've deleted this class with Spring config - whole config is in application-context.XML

**Comment by schaffe** [comment]
Ok, able to merge

**Pull request merged**